CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ that users understand how the changes affect the new version.
 
 version 2.1.0-dev
 -----------------
++ The scatter regions has been removed. Regions for scattering need to be 
+  provided externally. This is more efficient for pipelines that work on 
+  multiple samples.
```

Comment:
I feel like this warrants the version to become 3.0.0

Reply:
Agreed :+1: